sample/https-client.c

```diff
@@ -181,6 +180,30 @@ static int cert_verify_callback(X509_STORE_CTX *x509_ctx, void *arg)
 		return 0;
 	}
 }
+
+#ifdef _WIN32
+static void
+add_cert_for_store(X509_STORE *store, const char *name)
+{
+	HCERTSTORE sys_store = NULL;
+	PCCERT_CONTEXT ctx = NULL;
+
+	sys_store = CertOpenSystemStore(0, name);
+	if (!sys_store) {
+		return;
+	}
+	while ((ctx = CertEnumCertificatesInStore(sys_store, ctx))) {
+		X509 *x509 = d2i_X509(NULL, (unsigned char const **)&ctx->pbCertEncoded,
+			ctx->cbCertEncoded);
+		if (x509) {
+			if (X509_STORE_add_cert(store, x509) == 1) {
+				fputs("Add cert to store successfully\n", stdout);
```

Comment:
[too much noise](http://www.linfo.org/rule_of_silence.html), just return an error if any from `add_cert_for_store` and fail in the caller